src/server/http/setup_connection.js

```diff
@@ -3,6 +3,8 @@ import { map } from 'lodash';
 import secureOptions from './secure_options';
 
 export default function (kbnServer, server, config) {
+  const proxy = kbnServer && kbnServer.proxy;
```

Comment:
I definitely don't want to expose `proxy` property on the `kbnServer`, but I haven't figured out the better way yet.... Do you have any ideas?

Reply:
We've already got tons of things on it, so I think it's okey. We could probably rename this to be clearer. `newPlatformProxyListener` or something like that?

Reply:
✔️